env0/data_project.go

```diff
@@ -77,3 +67,32 @@ func dataProjectRead(ctx context.Context, d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func getProjectByName(name interface{}, meta interface{}) (client.Project, diag.Diagnostics) {
+	apiClient := meta.(*client.ApiClient)
+	projects, err := apiClient.Projects()
+	var project client.Project
+
+	if err != nil {
+		return client.Project{}, diag.Errorf("Could not query project by name: %v", err)
+	}
+	for _, candidate := range projects {
```

Comment:
If we go by name, we should throw if there's more than one project matching the name

Reply:
https://github.com/env0/terraform-provider-env0/pull/42/commits/00a2a8b6117b316725800e470b04b4f63f690ce9